packages/ui/tailwind-design-system.ts

```diff
@@ -276,20 +276,6 @@ export default {
             lime: 'var(--canary-label-background-lime-01)'
           }
         },
-        topbar: {
```

Comment:
🔥

Reply:
thanks for doing this @ankormoreankor ! We need to remove all other older tailwind classes like this